contracts/contracts/buyback/Buyback.sol

```diff
@@ -84,18 +87,35 @@ contract Buyback is Governable {
     }
 
     /**
-     * @dev Execute a swap of OGN for OUSD via Uniswap or Uniswap compatible
+     * @dev Execute a swap of OGV for OUSD via Uniswap or Uniswap compatible
      * protocol (e.g. Sushiswap)
      **/
     function swap() external onlyVault nonReentrant {
```

Comment:
We can remove the `onlyVault` and `nonReentrant` modifiers from this, since this function no does no actions. 

This also means the contract does not have to know who the vault is.

Reply:
Ah, yes. Have changed it, also removed all references to vault entirely